src/libraries/Microsoft.PowerFx.Connectors/Execution/OpenApiJsonSerializer.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.PowerFx.Connectors.Execution
+{
+    internal class OpenApiJsonSerializer
+    {
+        private readonly Dictionary<string, object> _dic;
+        private readonly MemoryStream _stream;
+        private readonly Utf8JsonWriter _writer;
+        private readonly OpenApiSchema _schema;
+
+        internal OpenApiJsonSerializer(OpenApiSchema schema, Dictionary<string, object> dictionary)
```

Comment:
>Dictionary<string, object> dictionary [](http://example.com/codeflow?start=61&length=37)

Just pass the FormulaValue directly. 
This lets you avoid FV.ToObject() and creating the intermediate objects.